go/nbs/block_store_test.go

```diff
@@ -71,8 +75,25 @@ func (suite *BlockStoreSuite) TestChunkStorePut() {
 }
 
 func (suite *BlockStoreSuite) TestChunkStorePutMany() {
-	input1, input2 := "abc", "def"
-	c1, c2 := chunks.NewChunk([]byte(input1)), chunks.NewChunk([]byte(input2))
+	input1, input2 := []byte("abc"), []byte("def")
+	c1, c2 := chunks.NewChunk(input1), chunks.NewChunk(input2)
+	suite.store.PutMany([]chunks.Chunk{c1, c2})
+
+	suite.store.UpdateRoot(c1.Hash(), suite.store.Root()) // Commit writes
+
+	// And reading it via the API should work...
+	assertInputInStore(input1, c1.Hash(), suite.store, suite.Assert())
+	assertInputInStore(input2, c2.Hash(), suite.store, suite.Assert())
+	if suite.putCountFn != nil {
+		suite.Equal(2, suite.putCountFn())
+	}
+}
+
+func (suite *BlockStoreSuite) TestChunkStorePutMoreThanMemTable() {
```

Comment:
Is the idea of this test that it will force a compaction? If so, would it make sense to assert the number of tables?